access_user.go

```diff
@@ -0,0 +1,382 @@
+	AccountID          string                                                `json:"account_id"`
+	AuthStatus         string                                                `json:"auth_status"`
+	CommonName         string                                                `json:"common_name"`
+	DevicePosture      map[string]AccessUserSingleActiveSessionDevicePosture `json:"devicePosture"`
+	DeviceID           string                                                `json:"device_id"`
+	DeviceSessions     map[string]AccessUserSingleActiveSessionDeviceSession `json:"device_sessions"`
+	Email              string                                                `json:"email"`
+	Geo                AccessUserSingleActiveSessionGeo                      `json:"geo"`
+	Iat                int64                                                 `json:"iat"`
+	IDP                AccessUserSingleActiveSessionIDP                      `json:"idp"`
+	IP                 string                                                `json:"ip"`
+	IsGateway          bool                                                  `json:"is_gateway"`
+	IsWarp             bool                                                  `json:"is_warp"`
+	MTLSAuth           AccessUserSingleActiveSessionMTLSAuth                 `json:"mtls_auth"`
+	ServiceTokenID     string                                                `json:"service_token_id"`
+	ServiceTokenStatus bool                                                  `json:"service_token_status"`
+	UserUUID           string                                                `json:"user_uuid"`
+	Version            int                                                   `json:"version"`
+	IsActive           bool                                                  `json:"isActive"`
+}
+
+type AccessUserSingleActiveSessionDevicePosture struct {
+	Check       AccessUserSingleActiveSessionDevicePostureCheck `json:"check"`
+	Data        map[string]interface{}                          `json:"data"`
+	Description string                                          `json:"description"`
+	Error       string                                          `json:"error"`
+	ID          string                                          `json:"id"`
+	RuleName    string                                          `json:"rule_name"`
+	Success     bool                                            `json:"success"`
+	Timestamp   string                                          `json:"timestamp"`
+	Type        string                                          `json:"type"`
+}
+
+type AccessUserSingleActiveSessionDevicePostureCheck struct {
+	Exists bool   `json:"exists"`
+	Path   string `json:"path"`
+}
+
+type AccessUserSingleActiveSessionDeviceSession struct {
+	LastAuthenticated int64 `json:"last_authenticated"`
+}
+
+type AccessUserSingleActiveSessionGeo struct {
+	Country string `json:"country"`
+}
+
+type AccessUserSingleActiveSessionIDP struct {
+	ID   string `json:"id"`
+	Type string `json:"type"`
+}
```

Comment:
There are instances around device posture checks where types are duplicated , these will be simplified , names are over-verbose for now.